acs-integration-tests/src/test/java/com/ge/predix/acceptance/test/ACSAcceptanceIT.java

```diff
@@ -79,74 +72,51 @@
     private PolicyHelper policyHelper;
 
     @Autowired
-    private ZacTestUtil zacTestUtil;
-
-    @Autowired
-    private Environment env;
-
-    @Autowired
     private ACSRestTemplateFactory acsRestTemplateFactory;
 
-    @Value("${ZONE1_NAME:testzone1}")
-    private String acsZone1Name;
-
-    @Value("${ZONE2_NAME:testzone2}")
-    private String acsZone2Name;
+    @Autowired(required = false)
+    private SetupUtility zacHelperUtil;
 
-    @Value("${ZONE3_NAME:testzone3}")
-    private String acsZone3Name;
-
-    @Value("${ACS_UAA_URL}")
+    @Value("${ZONE1_UAA_URL}")
     private String uaaUrl;
 
     private OAuth2RestTemplate acsAdminRestTemplate;
-    private boolean registerWithZac;
+    private HttpHeaders headersWithZoneSubdomain = new HttpHeaders();
+    private NewUaaTestUtil uaaTestUtil;
+    private String acsZoneName;
 
     @BeforeClass
-    public void setup() throws JsonParseException, JsonMappingException, IOException {
-        if (Arrays.asList(this.env.getActiveProfiles()).contains("public")) {
-            setupPublicACS();
-        } else {
-            setupPredixACS();
-        }
-        this.headersWithZoneSubdomain = new HttpHeaders();
-        this.headersWithZoneSubdomain.set("Predix-Zone-Id", this.testZoneSubdomain);
-    }
+    public void setup() throws IOException {
+        this.acsZoneName = this.zoneHelper.getRandomZoneName(this.getClass().getSimpleName());
+        this.headersWithZoneSubdomain.set("Predix-Zone-Id", acsZoneName);
+        this.uaaTestUtil = new NewUaaTestUtil(this.acsRestTemplateFactory, this.uaaUrl);
 
-    private void setupPredixACS() throws JsonParseException, JsonMappingException, IOException {
-        this.zacTestUtil.assumeZacServerAvailable();
+        this.acsAdminRestTemplate = this.uaaTestUtil
+                .createAcsAdminClientAndGetTemplate(Collections.singletonList(this.acsZoneName));
 
-        this.acsAdminRestTemplate = this.acsRestTemplateFactory.getACSTemplateWithPolicyScope();
-        this.registerWithZac = true;
-        this.testZoneSubdomain = this.zoneHelper
-                .createTestZone(this.acsAdminRestTemplate, this.acsZone1Name, this.registerWithZac).getSubdomain();
-    }
+        this.zoneHelper.createTestZone(this.acsAdminRestTemplate, this.acsZoneName, false);
```

Comment:
My thought was that this will be completely removed once we change all other tests. The OptionalTestSetup will do all Zac registration and zone helper doesn't need any knowledge of it.